semantic_conventions/http-common.yaml

```diff
@@ -9,6 +9,16 @@ groups:
         requirement_level: required
         brief: 'HTTP request method.'
         examples: ["GET", "POST", "HEAD"]
+        note: |
+          The HTTP request method usually contains one of a standard methods defined in [RFC9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-methods)
+          or [RFC5789](https://www.rfc-editor.org/rfc/rfc5789.html).
+
+          Instrumentations SHOULD call into TODO advice API (when available) before recording any telemetry
```

Comment:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-advice

Most likely advice will be part of the instrument creation parameters, so "call into ... before recording ..." doesn't apply well. Consider rephrasing it to "Instrumentations are RECOMMENDED to provide a list of well-known HTTP request methods via [Instrument advice](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#instrument-advice) (when available) to avoid the [cardinality explosion](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/supplementary-guidelines.md#aggregation-temporality)".